### ISSUE TITLE
ValueConverter with case sensitivity column name problem

### ISSUE THREAD

**Issue by Barbedx** [borisdj/EFCore.BulkExtensions#154]
Hi!
There are some bug with Value converter #47 
If we add to context field like this 
```
modelBuilder.Entity<Info>().Property(e => e.ConvertedTimeSpan)
                .HasColumnName("convertedTimeSpan")
                .HasConversion(new TimeSpanToTicksConverter());
```
We cant get them ValueConverter from dictionary, becouse is method are case sensitive
Look in enumerator of ObjectReaderEx:

```
        public override object this[string name]
        {
            get
            {
                ValueConverter converter;
                if (shadowProperties.Contains(name))
                {
                    var current = this.current.GetValue(this);
                    return context.Entry(current).Property(name).CurrentValue;
                }
                else 
                if (convertibleProperties.TryGetValue(name, out converter)) //Cant get name becouse is case sensitive
                {
                    var current = this.current.GetValue(this);
                    var currentValue = context.Entry(current).Property(name).CurrentValue;
                    return converter.ConvertToProvider(currentValue);
                }
                return base[name];
            }
        }
```

**Comment by borisdj** [comment]
I can confirm the issue.
The problem is not case sensitivity, but any difference between Property and Column Name when also having ValueConverter.
In addition I'm not sure how to fix this since convertibleProperties contains only column Names while 'name' is from property.
For now you can try to use the same Name for both Property and Column.

**Comment by borisdj** [comment]
Fixed with 2.4.1
